### PR TITLE
attempt to make it more stable as it seems to be related to not reall…

### PR DESCRIPTION
**Pull request by mswiderski**
…y pool data source

### CODE REVIEW

File: jbpm-test-coverage/src/test/java/org/jbpm/test/functional/jobexec/AsyncTaskTransactionTest.java [hunk 1]
```diff
@@ -154,4 +156,20 @@ private KieSession registerAsyncHandler(KieSession ksession) {
...
+        Properties dsProps = PersistenceUtil.getDatasourceProperties();
+        String jdbcUrl = dsProps.getProperty("url");
+        String driverClass = dsProps.getProperty("driverClassName");        
+
+        // Setup the datasource
+        PoolingDataSource ds1 = PersistenceUtil.setupPoolingDataSource(dsProps, "jdbc/jbpm-ds", false);
+        if (driverClass.startsWith("org.h2")) {
+            ds1.getDriverProperties().setProperty("url", jdbcUrl);
+        }
+        ds1.getDriverProperties().setProperty("POOL_CONNECTIONS", "false");
```
Comment (MarianMacik) [inline jbpm-test-coverage/src/test/java/org/jbpm/test/functional/jobexec/AsyncTaskTransactionTest.java]: Shouldn't we make this default until there is a real pooling datasource?

Comment (MarianMacik) [reply jbpm-test-coverage/src/test/java/org/jbpm/test/functional/jobexec/AsyncTaskTransactionTest.java]: But maybe it is not worth it if other tests are OK :)

Comment (mswiderski) [reply jbpm-test-coverage/src/test/java/org/jbpm/test/functional/jobexec/AsyncTaskTransactionTest.java]: that's exactly the issue :)

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins retest this

**Pull request merged**